static/app/components/workflowEngine/form/control/index.stories.tsx

```diff
@@ -18,17 +17,7 @@ export default storyBook('Form Controls', story => {
 
       <Form hideFooter>
         <Flex column gap={space(2)}>
-          <FormSection
-            title="Prioritize"
-            description="Update issue priority when the following thresholds are met:"
-          >
```

Comment:
this change is confusing, might be a bad merge?

Reply:
The `FormSection` was a component that was only used in this story, to format the `PriorityControl`. In production, `PriorityControl` is used _without_ `FormSection`, so I removed it.

The alternative would’ve been to either:
- inline `FormSection` into this story, but it’s 3 styled components + the exported one.
- move it to a place / rename it so that it’s seen as a “test util” - a file that we have to support test/story code. There is currently no structure in the codebase for this. We sometimes have `__stories__` directories, but we have to decide on how to do this first.

So here, I just opted to remove this component because the only thing it does is style some things for the story, and `PriorityControl` still looks fine in the story without it.